administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -23,13 +24,31 @@ const BirthdayForm = ({
   const [touched, setTouched] = useState(false)
   const { birthday } = value
   const showErrorMessage = touched || showRequired
+  const projectConfig = useContext(ProjectConfigContext)
+
+  const isBirthdayHintEnabled = (): boolean => {
```

Comment:
🔧 
```suggestion
  const showBirthdayHint = (): boolean => {
```
I think "enabled" is not a good naming, since it may be enabled by the projectConfig but not shown because the age conditions are false